### DISCUSSION TITLE
Geocoding errors?

### DISCUSSION THREAD

**Discussion by StyXman** [osm-search/Nominatim#2791] [Q&A]
A few days back a colleague in a community found a few examples of what we see as issues, but we're not sure if they are tagging or nominatim issues.

First example is https://www.openstreetmap.org/search?whereami=1&query=-35.97539%2C-62.73555#map=19/-35.97539/-62.73555 The search coords are clearly closer to Fray Justo ... de Oro than to Avellaneda, but nominatim gives an address composed by the building number on the first and the street name of the second.

The second one is related to case sensitivity. https://www.openstreetmap.org/search?query=avellaneda+350%2C+Trenque+Lauquen#map=19/-35.97971/-62.73278 gives the right address and the map shows fine, but the map in https://www.openstreetmap.org/search?query=avellaneda+350%2C+trenque+lauquen#map=19/-35.97280/-62.73710 is in a completely different place.

Are this related? Are they nominatim, OSM-web or tagging errors?

**Comment by lonvia** [comment]
There is clearly a bug in parenting of interpolations. https://www.openstreetmap.org/way/227495084 should be parented with Fray Justo ... de Oro, its start and end address nodes say so.

The second example has nothing to do with case sensitivity.  You are simply focused on different parts of the map which gives a different location bias.

**Reply by lonvia** [reply]
I've created issue #2797.

**Reply by lonvia** [reply]
All fixed now on the OSM servers.

**Reply by StyXman** [reply]
thanks a lot :)
